File: Tools/Simulation/StoffPy/loadShape.py
from physicsBox import PhysicsBox
from config import config
from geometry_tools import *

def LoadShape(fp):
	b = PhysicsBox(fp)

	for _ in range(0, config()["simulation_steps"]):
		b.simulation_step()
	
	b.filter_expulsed_points()
	p = b.to_plane()
	p.triangulate()
	p.render()

LoadShape("test.json")

File: Sewing/Tools/Simulation/StoffGo/shapes2D/load_shape.go
package shapes2d

import (
	"stoffgo/config"
)

func LoadShape(file string) *Plane {
	b := LoadShapeIntoBox(file)
	for i := 0; i < config.C.Simulation2D.SimulationSteps; i++ {
		b.SimulationStep()
	}
	b.FilterExpulsedPoints()
	p := b.ToPlane()
	p.Triangulate()
	return p
}
